examples/acquiring_data/querying_the_GOES_event_list.py

```diff
@@ -22,34 +23,34 @@
 # We then use the client to query the HEK for a list of events
 # that were detected by the GOES X-ray Sensor (XRS) instrument between
 # `tstart` and `tend`.
-result = client.search(hek.attrs.Time(tstart, tend),
+result = client.search(a.Time(tstart, tend),
                        hek.attrs.EventType(event_type),
                        hek.attrs.OBS.Observatory == 'GOES')
 
 ###################################################################
-# The result is returned as a `~sunpy.net.hek.hek.HEKTable`.
+# The result is returned as a `~sunpy.net.hek.hek.HEKResponse`.
 # We can print the number of flares and inspect the result information.
 # We can also print the key-values that correspond to the HEK parameters
 # returned in result.
 print(len(result))
 print(result)
-print(result.keys())
+print(result.table.keys())
```

Comment:
Does the table not have a columns or colnames?

Reply:
Yes, but should I change it to `table.colnames`?

Reply:
Leave it as keys if they produce the same output. It's shorter at least.

Reply:
yes, output is same though.